pkg/minishift/registration/redhat.go

```diff
@@ -55,26 +55,7 @@ func (registrator *RedHatRegistrator) Register(param *RegistrationParameters) er
 		return err
 	} else {
 		if strings.Contains(output, "not registered") {
-
-			//Configure subscription-manager for proxy enviornments
-			if param.ProxyServer != "" {
-				configCommand := fmt.Sprintf("sudo subscription-manager config ")
-				configCommand = configCommand +
-					fmt.Sprintf("--server.proxy_hostname %s "+
-						"--server.proxy_port %s ", param.ProxyServer, param.ProxyServerPort)
-				if param.ProxyUsername != "" {
-					configCommand = configCommand +
-						fmt.Sprintf("--server.proxy_user %s ", param.ProxyUsername)
-					if param.Password != "" {
-						configCommand = configCommand +
-							fmt.Sprintf("--server.proxy_password %s ", param.ProxyPassword)
-					}
-				}
-				if _, err := registrator.SSHCommand(configCommand); err != nil {
-					return err
-				}
-			}
-			subscriptionCommand := fmt.Sprintf("sudo subscription-manager register --auto-attach "+
+			subscriptionCommand := fmt.Sprintf("sudo -E subscription-manager register --auto-attach "+
```

Comment:
"-E" is used to preserve the environment variable for the shell we need it here so our proxy env variable can be picked easily.